runtime/ops/tty.rs

```diff
@@ -195,20 +194,13 @@ fn op_isatty(state: &mut OpState, rid: ResourceId) -> Result<bool, AnyError> {
   Ok(isatty)
 }
 
-#[derive(Serialize)]
-struct ConsoleSize {
-  columns: u32,
-  rows: u32,
-}
-
-#[op]
+#[op(v8)]
 fn op_console_size(
   state: &mut OpState,
   rid: ResourceId,
-) -> Result<ConsoleSize, AnyError> {
-  super::check_unstable(state, "Deno.consoleSize");
-
-  let size = StdFileResource::with_file(state, rid, move |std_file| {
+  out_buf: &mut [u32],
+) -> Result<(), AnyError> {
+  StdFileResource::with_file(state, rid, move |std_file| {
```

Comment:
I'm not sure it makes sense to provide stdout and stderr here because if those are piped there doesn't seem to be a way to get the console screen size. I think we should think more about how this api would work with eventual pty support and maybe not stabilize it atm.